src/utils/misc-utils.ts

```diff
@@ -1,10 +1,20 @@
+import { ConnectorTag } from 'components/capture/create';
+
 export const stripPathing = (stringVal: string) => {
     return stringVal.substring(
         stringVal.lastIndexOf('/') + 1,
         stringVal.length
     );
 };
 
+export const getConnectorName = (connectorTag: ConnectorTag) => {
+    if (connectorTag.title) return connectorTag.title;
+    if (connectorTag.connectors.detail) return connectorTag.connectors.detail;
+    if (connectorTag.image_tag) return stripPathing(connectorTag.image_tag);
+
+    throw new Error('Could not figure out Connector Name');
+};
```

Comment:
This will probably slowly grow - but this is the current order of what we want to show for capture name when querying the `Connectors_Tag` table.